### DISCUSSION TITLE
Welcome to kobalte Discussions!

### DISCUSSION THREAD

**Discussion by fabien-ml** [kobaltedev/kobalte#73] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by popuguytheparrot** [comment]
Hello

How will you deal with extra js code in chunks and bundles due to the use of compound components (namespace export) ?
